File: Pertemuan 5/Tugas5.py
nama = str(input("Masukan nama lengkap Anda = "))
ipk = str(input("Masukan IPk Anda = "))
studi = str(input("Masukan lama masa studi Anda = "))

if(float(ipk) >= 3.51 and float(ipk) <= 4 and float(studi) <= 4):
    print(nama + " Anda lulus dengan Pujian Tertinggi (Summa Cumlaude) dengan IPK " + ipk + " dengan masa studi " + studi + " tahun ")
elif(float(ipk) >= 3.51):
    print(nama + "Anda lulus dengan Pujian (Cumlaude) dengan IPK " + ipk)
elif(float(ipk) >= 3.01 and float(ipk) < 3.51):
    print(nama + "Anda lulus Sangat Memuaskan dengan IPK " + ipk)
elif(float(ipk) >= 2.76 and float(ipk) < 3.01):
     print(nama + "Anda lulus Memuaskan dengan IPK " + ipk)
elif(float(ipk) >= 2.00 and float(ipk) < 2.76):
     print(nama + "Anda lulus Cukup Memuaskan dengan IPK " + ipk)
elif(float(ipk) < 2.00):
    print(" Maaf " + nama + "Anda tidak dengan IPK " + ipk)
elif(float(ipk) > 4.00 and float(ipk) < 0.00):
    print("IPK yang Anda input tidak valid ")

File: Pertemuan 5/Tugas5.java
import java.util.Scanner;

public class Tugas5 {

    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Nama lengkap Anda : ");
        String namaLengkap = scanner.nextLine();

        System.out.print("Nilai IPK Anda : ");
        double ipk = scanner.nextDouble();

        System.out.print("Lama masa studi Anda : ");
        int lamaStudi = scanner.nextInt();

        System.out.println();
        System.out.println("Predikat Kelulusan Tahun 2021");
        System.out.println("Nama  : " + namaLengkap);
        System.out.println("IPK : " + ipk);
        System.out.println("Lma masa studi : " + lamaStudi);
    
        if (ipk >= 3.51 && ipk <= 4.00 && lamaStudi <= 4) {
            System.out.println("Selamat! Anda lulus dengan Pujian Tertinggi (Summa Cumlaude).");
        } else if (ipk >= 3.51 && ipk <= 4.00) {
            System.out.println("Selamat! Anda lulus dengan Pujian (Cumlaude).");
        }  else if (ipk >= 3.01 && ipk < 3.51) {
            System.out.println("Selamat! Anda lulus dengan Sangat Memuaskan.");
        } else if (ipk >= 2.76 && ipk < 3.01) {
            System.out.println("Selamat! Anda lulus dengan Memuaskan.");
        } else if (ipk >= 2.00 && ipk < 2.76) {
            System.out.println("Selamat! Anda lulus dengan Cukup.");
        } else if (ipk < 2.00) {
            System.out.println("Maaf, Anda dinyatakan Tidak Lulus.");
            System.out.println("Tetap Semangat, Jangan Putus Asa!");
        } else {
            System.out.println("Maaf, IPK Anda Tidak Valid");
        }
    scanner.close();
    }
}